### DISCUSSION TITLE
Discord Community invite link is invalid

### DISCUSSION THREAD

**Discussion by richardkmichael** [cypress-io/cypress#25097] [Questions and Help] [answered]
On the [support page](https://docs.cypress.io/guides/references/troubleshooting#Support-channels), the Discord Invite link is invalid.

Is there an updated one somewhere else?  Does the Discord Community still exist?

**Comment by jjhelguero** [comment, marked as answer]
Here is the updated link https://discord.gg/cypress

**Reply by emilyrohrbough** [reply]
& it looks like @jjhelguero opened a PR to fix the link: https://github.com/cypress-io/cypress-documentation/pull/4928
